src/honeycomb/opentelemetry/distro.py

```diff
@@ -20,10 +35,13 @@ def configure_opentelemetry(
     Configures the OpenTelemetry SDK to send telemetry to Honeycomb.
 
     Args:
-        options (HoneycombOptions, optional): the HoneycombOptions used to
-        configure the the SDK
+        options (HoneycombOptions, optional*): the HoneycombOptions used to
+        configure the the SDK. These options can be set either as parameters
+        to this function or through environment variables
+
+        *API key is a required option.
```

Comment:
```suggestion
        Note: API key is a required option.
```

Can we change this to a note instead of an asterisk? It feels like a syntax thing in code context.